### ISSUE TITLE
Log level fine-tuning

### ISSUE THREAD

**Issue by rokf** [p4tin/goaws#230]
Hey, I propose that some kind of log level configuration logic is added to at least the `gosqs` part of the application.

There are two logs that are  especially bothering me:

https://github.com/p4tin/goaws/blob/6ecd9bf473ad2b2a88dba5224957826665710dec/app/gosqs/gosqs.go#L401

https://github.com/p4tin/goaws/blob/6ecd9bf473ad2b2a88dba5224957826665710dec/app/gosqs/gosqs.go#L451

As the SQS instance is waiting for messages it is spamming out useless logs every second. This can be costly in certain deployment scenarios.

**How about introducing an error log level?**

I've also noticed that the flag configuration logic in https://github.com/p4tin/goaws/blob/7aca39ed52f1b6b5ad83767a5dd6dc320fea8c8d/app/cmd/goaws.go#L22 is mentioning that the warning level is the default log level, however that is not true - it's the info level.

**Comment by Admiral-Piett** [comment]
Good point `rokf` no reason to print anything at this point.  We can make these debug level logs, and clean up the help logic.

**Comment by attilah** [comment]
@Admiral-Piett I came for the very same thing ;-) looking for significant log entries (there IS a message) is hard in an ever scrolling window - which I have no control over.

**Comment by Admiral-Piett** [comment]
Yeah I think I got rid of most of the prints in the 0.5.0 release, but I still need to sweep the code for the rest.  Sorry about that.  If there are some that are particularly troublesome let me know?
